frontend/src/views/ShootList.vue

```diff
@@ -490,6 +490,14 @@ export default {
           defaultSelected: true,
           hidden: this.projectScope || !this.isAdmin
         },
+        {
+          text: 'Control Plane HA',
```

Comment:
<img width="658" alt="image" src="https://user-images.githubusercontent.com/1574023/213686782-98f3391c-dd1e-4623-8783-9273b03330c2.png">
```suggestion
          text: 'CONTROL PLANE HIGH AVAILABILITY',          
```
or maybe only `HIGH AVAILABILITY`

Reply:
I go for `HIGH AVAILABILITY` as the column gets too wide otherwise